From 9b6446b15771822edb28f4624aa54c10d3c5983b Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robux4@ycbcr.xyz>
Date: Thu, 30 Jan 2025 06:53:18 +0100
Subject: [PATCH] crt: add missing DXVAHD_CreateDevice for x86

It was first added to arm32 in 4a306baa4a206c9314bdc129e98abfa1aebe2b0e.
Then to the common one in 96fe9ec413001bfc3f5971df64bcdedc216a9020.
But it was always missing in x86.

I verified that's the only one missing in that .def.
---
 mingw-w64-crt/lib32/dxva2.def | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mingw-w64-crt/lib32/dxva2.def b/mingw-w64-crt/lib32/dxva2.def
index d43af5be3..ea343a53f 100644
--- a/mingw-w64-crt/lib32/dxva2.def
+++ b/mingw-w64-crt/lib32/dxva2.def
@@ -8,6 +8,7 @@ EXPORTS
 CapabilitiesRequestAndCapabilitiesReply@12
 DXVA2CreateDirect3DDeviceManager9@8
 DXVA2CreateVideoService@12
+DXVAHD_CreateDevice@20
 DegaussMonitor@4
 DestroyPhysicalMonitor@4
 DestroyPhysicalMonitors@8
-- 
2.45.2

